src/cli/api/local_api.py

```diff
@@ -46,9 +43,18 @@ def __is_application_unique(self, application: str) -> bool:
         Args:
             application: application name
         """
-
         return not self.__config_manager.application_exists(application)
 
+    def list_applications(self) -> List[ApplicationType]:
+        """
+        Function to list the local applications
```

Comment:
Not an error but: all the references to local are not needed because we are in local_api.py

Reply:
Removed local text